view/theme/frio/css/style.css

```diff
@@ -417,6 +417,10 @@ nav.navbar {
     z-index: 1030;
     color: $nav_icon_color;
 }
+#topbar-first .navbar-toggle {
+    margin-top: 5px;
+    margin-bottom: 0; /* adding a bottom margin is useless */
```

Comment:
Maybe, but why? If you leave a comment, please be as informative as possible.

Reply:
Oh, to aligh the search icon and the three dot icons properly. Right now they use bootstrap margins, that is why they aren't aligned.

Reply:
So removing the margin is purposeful and the comment isn't warranted, is it?

Reply:
Is this comment still relevant/necessary in light of your elaboration?